aiodns/__init__.py

```diff
@@ -6,7 +6,14 @@
     import trollius as asyncio
 import pycares
 
-from . import error
+from typing import (
```

Comment:
Those comments are evaluated by the tools performing the type checking (like mypy).